Gemfile.lock

```diff
@@ -120,17 +132,46 @@ GEM
     mini_portile2 (2.4.0)
     minitest (5.13.0)
     msgpack (1.3.1)
+    msgpack (1.3.1-java)
+    msgpack (1.3.1-x64-mingw32)
+    msgpack (1.3.1-x86-mingw32)
+    multi_json (1.14.1)
+    multi_xml (0.6.0)
+    multipart-post (2.1.1)
     nio4r (2.5.2)
+    nio4r (2.5.2-java)
     nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
+    nokogiri (1.10.7-java)
+    nokogiri (1.10.7-x64-mingw32)
+      mini_portile2 (~> 2.4.0)
+    nokogiri (1.10.7-x86-mingw32)
+      mini_portile2 (~> 2.4.0)
+    oauth2 (1.4.2)
+      faraday (>= 0.8, < 2.0)
+      jwt (>= 1.0, < 3.0)
+      multi_json (~> 1.3)
```

Comment:
The different dependencies shown in the Gemfile.lock are based on each Ruby Gem individually. This will not cause errors (can confirm, app not broken on my system and all tests passing). The formatting changes are due to Rubocops linting.

Reply:
Okay I have no idea what to do. Overall this is a classic example of Dependency Hell. We have one part of our app that wants `multi_json ~> 1.3` and another that wants `multi_json 1.14.1`. This may not cause a problem now but may down the road. As evidenced by the fact that it works for both Shane and myself, but in personal websites I have done this has led to problems (ie it worked for features at the time but new features led to problems). I am also a little concerned by the fact that the most current release of multi_json is 1.14.1 so just updating that wont work. As far as I can tell oauth2 is where this version of multi_json is coming from. (https://github.com/oauth-xx/oauth2/blob/v1.4.2/oauth2.gemspec) Which is needed I guess for omniauth-facebook. While my gut feeling says this wont matter. My logical side says this is not a pressing concern and should wait till we can get an adult to review. :)

Reply:
Well frack me I am an idiot. 1.14 is greater then 1.3.